tests/comparisons/Diff/the_diff_pgsql.php

```diff
@@ -0,0 +1,193 @@
+<?php
+use Migrations\AbstractMigration;
+
+class TheDiffPgsql extends AbstractMigration
+{
+
+    public function up()
+    {
+        $this->table('articles')
+            ->dropForeignKey([], 'articles_user_id')
+            ->removeIndexByName('unique_slug')
+            ->removeIndexByName('rating_index')
+            ->removeIndexByName('by_name')
+            ->update();
+
+        $this->table('articles')
+            ->removeColumn('content')
+            ->changeColumn('title', 'text')
+            ->changeColumn('name', 'string', [
+                'length' => 50,
+            ])
+            ->update();
+
+        $table = $this->table('categories');
+        $table
+            ->addColumn('name', 'string', [
+                'default' => null,
+                'limit' => 255,
+                'null' => false,
+            ])
+            ->addColumn('user_id', 'integer', [
+                'default' => null,
+                'limit' => 10,
+                'null' => false,
+            ])
+            ->addIndex(
+                [
+                    'user_id',
+                ]
+            )
+            ->addIndex(
+                [
+                    'name',
+                ]
+            )
+            ->create();
+
+        $this->table('categories')
+            ->addForeignKey(
+                'user_id',
+                'users',
+                'id',
+                [
+                    'update' => 'RESTRICT',
+                    'delete' => 'RESTRICT'
+                ]
+            )
+            ->update();
+
+        $this->table('articles')
+            ->addColumn('category_id', 'integer', [
+                'default' => null,
+                'length' => 10,
+                'null' => false,
+            ])
+            ->update();
+
+        $this->table('articles')
+            ->addIndex(
+                [
+                    'category_id',
+                ],
+                [
+                    'name' => 'category_id',
+                ]
+            )
+            ->addIndex(
+                [
+                    'slug',
+                ],
+                [
+                    'name' => 'unique_slug',
+                ]
+            )
+            ->addIndex(
+                [
+                    'name',
+                ],
+                [
+                    'name' => 'rating_index',
+                ]
+            )
+            ->update();
+
+        $this->table('articles')
+            ->addForeignKey(
+                'category_id',
+                'categories',
+                'id',
+                [
+                    'update' => 'restrict',
+                    'delete' => 'restrict'
```

Comment:
Why is it `RESTRICT` in uppercase a few lines above, but here it is in lowercase?